department-of-veterans-affairs/next-build
# Update sitemap a11y scan

## Description
Relates to #[16117](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16117).

## Testing done

Running test locally with all the options via CLI, e.g. `BASE_URL=https://va.gov USE_PROXY=false PW_WIDTH_VALUE=768 PW_HEIGHT_VALUE=720 PW_BROWSER_VALUE=firefox SEGMENT_INDEX=0 yarn playwright test --project=a11y`

## Screenshots

<img width="851" alt="Screenshot 2024-04-05 at 11 18 40 AM" src="https://github.com/department-of-veterans-affairs/next-build/assets/3640707/e3676e15-fadd-4c86-af85-36709bb5beda">

## QA steps

1. Go to the a11y workflow section on GH: https://github.com/department-of-veterans-affairs/next-build/actions/workflows/a11y.yml
2. Look at the latest run to see it is green.
3. Download the `a11y-scan-1` artifact and open it to confirm violations are listed by URL with a valid JSON syntax. The browser name and viewport size should be included with each URL listed.
4. Re-run the latest job and see it passes. The run should take around 20 minutes.
5. Repeat steps 2 and 3 to see they still pass.
